deploy.gradle

```diff
@@ -22,6 +22,11 @@ if (project.hasProperty('release') && jdkVersion != '8') {
 }
 
 subprojects {
+  if (it.name == 'iceberg-open-api') {
```

Comment:
Just wondering do we need `project.hasProperty('release')` check also here? Because without `-Prelease` it should work as usual?

Reply:
we also don't want to publish the artifact when running `./gradlew publishToMavenLocal`. Exiting early here means we basically don't run any publishing/signing tasks for this particular artifact